### PR TITLE
Mention default username and password

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

**Comment by tianon** [comment]
cc @Godin

File: sonarqube/content.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The server is started this way:
 $ docker run -d --name sonarqube -p 9000:9000 -p 9092:9092 %%IMAGE%%
 ```
 
+The [default username and password](https://docs.sonarqube.org/display/SONAR/Authentication) is `admin:admin`.
```
Comment (Godin) [inline sonarqube/content.md]: @simon04 nice addition! :thumbsup: To publish I think will be better to phrase this slightly differently
```
By default you can login as `admin` with password `admin`.
```
in order to clearly separate username and password.

**Comment by simon04** [comment]
Thanks for the review. I rephrased the sentence (but kept the URL to the SONAR docs).

**Comment by Godin** [comment]
@simon04 thanks for paraphrasing! :thumbsup:

**Pull request merged**